eslint.config

```diff
@@ -0,0 +1,12 @@
+{
+  "parserOptions": { "ecmaVersion": 6 },
+  "extends": "airbnb-base",
+    rules: {
+    'no-shadow': 0,
+    'no-param-reassign': 0,
+    'eol-last': 0,
+  },
+  "globals": {
+    "document": false,
+  }
+}
```

Comment:
This file is not necesary, the one that should be selected when runing `npx eslint --init` should be the one in JSON format so that we don't end up with two config files, as this might bring conflicts along the way, as we can see here 
![image](https://user-images.githubusercontent.com/46115725/73866113-e90bed80-4809-11ea-902d-0ae3749e69b0.png)
From the [Microverse's linter setup](https://github.com/microverseinc/linters-config/tree/master/javascript)